### PR TITLE
I2C connector documentation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +10,7 @@ CircuitPython supports [more than 200 different boards](https://circuitpython.or
 
 ![LILYGO T8](docs/T8.jpg)
 
-The planned adapterplate adds a 3-way button for input, place to hold the little 350 mAh battery and interface for I2C with regular 2.54mm 4pin JST XH as well as the small Stemma QT 1mm JST SH connector (QWIIC).
+The planned adapterplate adds a 3-way button for input, place to hold the little 500 mAh battery and interface for I2C with regular 2.54mm 4pin JST XH as well as the small Stemma QT 1mm JST SH connector (QWIIC).
```
Comment (kreier) [inline README.md:13]: might be 530 or 600 mAh in the 20mm between the 2 DIL connectors

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # ssis:bit
-AIO solution for SSIS students
+
+AIO microcomputer solution with CircuitPython, display and battery for SSIS students.
```
Comment (kreier) [inline README.md:3]: better description

File: README.md [hunk 3]
```diff
@@ -86,3 +170,22 @@ The __esptool.py__ is the software provided by esp to upload the firmware to the
...
+
+Some of these challenges are solved partially with the micro:bit - but limited in the end with just 2 buttons as input and a 5x5 red LED matrix as output. And the power adapter with 2 AAA batteries is not physically connected, making the solution less robust. Advantages with the Lilygo T8 ST7789:
+
+1. AIO solution with 3 button input, 240x135 color display output and small LiPo battery
+2. Standardaised QWIIC / STEMMA QT 1mm JST SH adapter for I2C sensors and IO
+3. Standardised STEMMA 2mm GROVE 2mm JST PH connector
+4. SD Card with GB of space for websites, data, images
+5. WIFI build in to serve as an AP students can connect to, or to connect to the Internet
+6. USB-C for charging and simple upload of programs to the CIRCUITPY folder
+7. REPL - get starting programming right away!
```
Comment (kreier) [inline README.md:191]: Good documentation of the motivation

File: docs/index.md [hunk 4]
```diff
@@ -10,7 +10,18 @@ CircuitPython supports [more than 200 different boards](https://circuitpython.or
...
+### Documentation by LilyGo or TTGO
+
+Parts of the documentation for this board can be found under both brand names LilyGo and TTGO. The main board resembles an [LilyGo T8](https://github.com/Xinyuan-LilyGO/LILYGO-T8) with a [ESP32-S2](https://github.com/Xinyuan-LilyGO/ESP32_S2) CPU but an added ST7789 display. So the schematic is found under [T-Display S2](https://github.com/Xinyuan-LilyGO/LilyGo-T-Display-S2) even though it is not sold as such. Here are some repositories for further informations:
+
+- https://github.com/Xinyuan-LilyGO/LilyGo-T-Display-S2
+- https://github.com/Xinyuan-LilyGO/LilyGO-T-DisplayGD32
+- https://github.com/Xinyuan-LilyGO/LILYGO-T-display-RP2040
+- https://github.com/Xinyuan-LilyGO/TTGO-T-Display
+- https://github.com/Xinyuan-LilyGO/LILYGO-T8
+- https://github.com/Xinyuan-LilyGO/ESP32_S2
```
Comment (ssisbit) [inline docs/index.md:24]: Good overview of 4 different repositories for the same board

File: README.md (hunk 3 above)
Comment (ssisbit) [reply README.md:191]: agreed

File: docs/index.md [hunk 5]
```diff
@@ -150,6 +171,22 @@ The __esptool.py__ is the software provided by esp to upload the firmware to the
...
+
+Some of these challenges are solved partially with the micro:bit - but limited in the end with just 2 buttons as input and a 5x5 red LED matrix as output. And the power adapter with 2 AAA batteries is not physically connected, making the solution less robust. Advantages with the Lilygo T8 ST7789:
+
+1. AIO solution with 3 button input, 240x135 color display output and small LiPo battery
+2. Standardaised QWIIC / STEMMA QT 1mm JST SH adapter for I2C sensors and IO
+3. Standardised STEMMA 2mm GROVE 2mm JST PH connector
+4. SD Card with GB of space for websites, data, images
+5. WIFI build in to serve as an AP students can connect to, or to connect to the Internet
+6. USB-C for charging and simple upload of programs to the CIRCUITPY folder
+7. REPL - get starting programming right away!
```
Comment (ssisbit) [inline docs/index.md:192]: Time will show how this supports the students

File: README.md [hunk 6]
```diff
@@ -24,27 +25,110 @@ Parts of the documentation for this board can be found under both brand names Li
...
 The ssis:bit runs [CircuitPython](https://circuitpython.org/) and can be easily programmed with the [Mu Editor](https://codewith.mu/en/). There is a lot to learn on [Adafruit with CircuitPython]() and with their great libraries there is less frustration getting things started.
 
-## Motivation and History
+## GPIO
 
-More will be updated in the [docs/history](docs/history.md) document, but the idea is from 2020 with the [T-Display](https://github.com/kreier/t-display) board. Similar idea: Not just the Microcomputer but included power supply (LiPo), display (just 240x135 color) and input device (not keyboard, but 3 buttons) to have it always with you and ready to use.
+This document from LilyGO documents the pins of the T8.
 
-A major challenge to get students use more of the microcomputers like Arduino, Raspberry Pi or micro:bit is the need for 3 more accessories to use them:
+![T8 pinmap](docs/T8-pinmap.jpg)
```
Comment (ssisbit) [inline README.md:36]: helpful information

File: README.md (hunk 1 above)
Comment (ssisbit) [reply README.md:13]: A time estimate and some experimental values should be added after some time.

**Review by ssisbit — COMMENTED** [review]
Good documentation, shrinked images and rationale behind the JST XH connector for I2C.

**Pull request merged**